### ISSUE TITLE
create list of domain merged media

### ISSUE THREAD

**Issue by hroberts** [mediacloud/backend#380]
generate a spreadsheet of proposed media merges for the single media per domain migration.

**Comment by rahulbot** [comment]
One of the first steps for #390

**Comment by hroberts** [comment]
I have generated a big spreadsheet with a list of every media source that would be merged by this move.:

https://drive.google.com/open?id=1cXeH26RD5KzUlFNzUV4KhJZdbvK3gozq2vzQsaFCWwI

This list include both media that would be merged because they have the same domain and media that would be merged because an existing medium is already marked as a duplicate of another.  The resulting list includes ~160k media over ~28k domains.

The first column of the spreadsheet is the domain generated from the media source url.  The dup_media_id field has the id of the parent media for media already manually marked as a duplicate of another media.  I have also include the total number of stories and sentences in each medium to get an idea of which sources include data and how much.

This spreadsheet is a good first step, but this is so much data (>160k media) that I think it will require some analytical queries to really make sense of the proposed changes.

As part of this work, I updated the media source url -> domain translation code.  One significant change is that I do not recognize any country specific editions of sites that are not represented in the host name itself.  This is because for this data model to work, we have to be able to identify a media source by a url.  For the sites that have a country edition that is only identified in the url path (eg. https://www.theguardian.com/au?INTCMP=CE_AU), there is no way to identify the country/edition of a given story from the url (eg. https://www.theguardian.com/world/2018/jun/08/kim-jong-un-impersonator-says-he-was-held-in-singapore).  

Some thoughts as I try to make sense of this, just to have them somewhere:
* we will probably need to manually review all potential collection membership changes generated by this change so that we know whether, for example, 
* we will likely want to merge feeds from duped media into the parent media source, but some validation of that feed merging process would make me more comfortable.
